frappe/public/scss/website/index.scss

```diff
@@ -9,6 +9,7 @@
 @import "../common/buttons";
 @import "../common/modal";
 @import "../desk/toast";
+@import "../desk/css_variables.scss";
```

Comment:
why is this needed? why not `../common/css_variables`?

If something is missing in `common/css_variables` and you want to use it for the website... you can move that variable from `desk/css_variables.scss` to `common/css_variables.scss`